tools/UpdateDotnetRuntime.ps1

```diff
@@ -210,32 +217,26 @@ if ($dotnetUpdate.ShouldUpdate) {
 
     Find-Dotnet
 
-    $feedname = if ($UseRTMFeed) {
-        'dotnet5-rtm'
-    } elseif ($UseNuGetOrg) {
-        'dotnet5'
-    } else {
-        'dotnet-internal'
-    }
+    $feedname = if ($UseNuGetOrg) { 'nuget.org' } elseif ($UseInternalFeed) { 'dotnet-internal' } else { 'dotnet' }
 
-    $addDotnet5Source = (-not (Get-PackageSource -Name $feedname -ErrorAction SilentlyContinue))
+    $addDotnetSource = (-not (Get-PackageSource -Name $feedname -ErrorAction SilentlyContinue))
 
-    if (!$UseNuGetOrg -and ($addDotnet5Source -or $addDotnet5InternalSource)) {
+    if (!$UseNuGetOrg -and ($addDotnetSource)) {
         $nugetFileSources = ([xml](Get-Content .\nuget.config -Raw)).Configuration.packagesources.add
 
-        if ($addDotnet5Source -and $feedname -ne 'dotnet-internal') {
-            $dotnet5Feed = $nugetFileSources | Where-Object { $_.Key -eq $feedname } | Select-Object -ExpandProperty Value
-            Register-PackageSource -Name $feedname -Location $dotnet5Feed -ProviderName NuGet
+        if ($addDotnetSource -and $feedname -ne 'dotnet-internal') {
+            $dotnetFeed = $nugetFileSources | Where-Object { $_.Key -eq $feedname } | Select-Object -ExpandProperty Value
+            Register-PackageSource -Name $feedname -Location $dotnetFeed -ProviderName NuGet
             Write-Verbose -Message "Register new package source $feedname" -verbose
         }
 
-        if ($addDotnet5Source -and $InteractiveAuth -and $feedname -eq 'dotnet-internal') {
+        if ($addDotnetSource -and $feedname -eq 'dotnet-internal') {
             # This NuGet feed is for internal to Microsoft use only.
-            $dotnet5InternalFeed = 'https://pkgs.dev.azure.com/dnceng/internal/_packaging/dotnet5-internal/nuget/v3/index.json'
-            $updatedNugetFile = (Get-Content .\nuget.config -Raw) -replace "</packageSources>", "  <add key=`"dotnet5-internal`" value=`"$dotnet5InternalFeed`" />`r`n  </packageSources>"
+            $dotnetInternalFeed = $dotnetMetadataJson.internalfeed.url
+            $updatedNugetFile = (Get-Content .\nuget.config -Raw) -replace "</packageSources>", "  <add key=`"dotnet6-internal`" value=`"$dotnetInternalFeed`" />`r`n  </packageSources>"
             $updatedNugetFile | Out-File .\nuget.config -Force
```

Comment:
Use `$PSScriptRoot/../nuget.config` or save that path to a variable and use that variable.

Reply:
fixed